pkg/models/object.go

```diff
@@ -45,6 +45,12 @@ const (
 	// ExternalEntityReferenceRuntime_Transformer is a "type" under runtime
 	// UIDs include: joinByField, organize, seriesToColumns, etc
 	ExternalEntityReferenceRuntime_Transformer = "transformer"
+
+	// ObjectStoreScopeEntity is organized in: {kind}/{uid}
+	ObjectStoreScopeEntity = "entity"
```

Comment:
this will make sense when looking at the router:
https://github.com/grafana/grafana/pull/56348/files#diff-ce50f456e6212a305e7bf52b8520494e742963a5123a1ac5987c2f7afd3b18a2R65